test/main.sh

```diff
@@ -132,9 +132,12 @@ export MICROCEPH_SNAP_CHANNEL
 MICROCLOUD_SNAP_CHANNEL="${MICROCLOUD_SNAP_CHANNEL:-latest/edge}"
 export MICROCLOUD_SNAP_CHANNEL
 
-MICROOVN_SNAP_CHANNEL="${MICROOVN_SNAP_CHANNEL:-latest/edge}"
+MICROOVN_SNAP_CHANNEL="${MICROOVN_SNAP_CHANNEL:-22.03/stable}"
```

Comment:
Do we still need this env variable as MicroOVN is now built from source?

Reply:
Yes, the recommended way to install a built snap is to install the default snap first and then use `--dangerous`. This is to preserve any snap interfaces that won't be set up normally with just `--dangerous` or `--devmode`.

Reply:
I see thanks!